packages/hash/api/src/model/ontology/property-type.model.ts

```diff
@@ -22,7 +22,7 @@ type PropertyTypeModelConstructorParams = {
  * @class {@link PropertyTypeModel}
  */
 export default class {
-  private propertyType: PropertyTypeWithMetadata;
+  propertyType: PropertyTypeWithMetadata;
```

Comment:
We probably do, I intended this commit to be temporary as I think Ben is going to unprotect them while changing the getter methods, but to avoid the PR being blocked I'll update it here as well

Reply:
That's been done